code/obj/vehicle.dm

```diff
@@ -2266,7 +2266,6 @@ obj/vehicle/forklift/attackby(var/obj/item/I, var/mob/user)
 	//attacking rider on forklift
 	if(rider && rider_visible && I.force)
```

Comment:
Extraneous check, since you're checking for it in the parent call.

Reply:
thanks, i didn't spot that